src/components/sync-sites-modal-selector.tsx

```diff
@@ -100,6 +100,31 @@ function SearchSites( {
 	);
 }
 
+// render unsupported sites at the bottom of the list
+const getSortedSites = ( sites: SyncSite[] ) => {
+	return [ ...sites ].sort( ( a, b ) => {
+		if ( a.syncSupport === 'unsupported' && b.syncSupport === 'unsupported' ) {
+			return 0;
+		}
+		if ( a.syncSupport === 'unsupported' ) {
+			return 1;
+		}
+		if ( b.syncSupport === 'unsupported' ) {
+			return -1;
+		}
+		if ( a.syncSupport === 'needs-transfer' && b.syncSupport === 'needs-transfer' ) {
+			return 0;
+		}
+		if ( a.syncSupport === 'needs-transfer' ) {
+			return 1;
+		}
+		if ( b.syncSupport === 'needs-transfer' ) {
+			return -1;
+		}
+		return 0;
```

Comment:
Is there a simpler approach for solving that sorting that would be easier to understand and maintain?

Reply:
It's classic js sort. I will look for some libraries with easier using at a glance

Reply:
Yep, updated the PR